app/src/main/java/de/westnordost/streetcomplete/quests/PriorityList.java

```diff
@@ -0,0 +1,27 @@
+package de.westnordost.streetcomplete.quests;
+
+import java.util.List;
+
+import de.westnordost.streetcomplete.view.Item;
+
+public class PriorityList {
+    public static List<Item> BuildList(List<Item> allItems, List<String> popularNames){
```

Comment:
According to usual Java naming conventions, functions always start with lowercase letters.

Reply:
Ops, fixed.